src/cli.ts

```diff
@@ -6,13 +6,14 @@ import {renderScreenshot} from "./index";
 
 const options = new Command("terminal-screenshot")
   .description("Render terminal ANSI output into images!")
-  .helpOption("-h, --help", "display usage help.")
-  .option("-d, --data [string]", "Data to be render to the terminal.")
-  .option("-m --margin [number]", "Margin to leave around the terminal area in pixels. (default: 0)")
-  .option("-f --font-family [string]", "Font family to use in terminal output. (default: Monaco)")
   .option("-b --background-color [css-color]", "Background color of the terminal. (default: black)")
```

Comment:
Given the capabilities of the new `xterm.Theme`, I wonder if we should also remove `--background-color` and `--margin`, and rely solely on the xterm theme configuration .. WDYT?

Reply:
I got some time today to experiment with it, and it looks like we have to keep both. `--background-color` is different than `ITheme.background`, as the former will be used in the `--border`, while the latter won't. So I take back my original suggestion.